Python/libraries/recognizers-number/recognizers_number/resources/german_numeric.py

```diff
@@ -144,11 +144,14 @@ def DoubleWithoutIntegralRegex(placeholder):
                               ("achtzig", 80),
                               ("neunzig", 90),
                               ("hundert", 100),
+                              ("einhundert", 100),
                               ("tausend", 1000),
+                              ("eintausend", 1000),
```

Comment:
Is this really needed?

Reply:
I added it because i though it would fix a failed test, but it does not solve it. I will remove it.